src/Sylius/Bundle/PayumBundle/DependencyInjection/Configuration.php

```diff
@@ -36,6 +36,9 @@ public function getConfigTreeBuilder(): TreeBuilder
         $rootNode
             ->addDefaultsIfNotSet()
             ->children()
+                ->arrayNode('gateway_config_validation_groups')
+                    ->variablePrototype()->end()
+                    ->useAttributeAsKey('name')->end()
```

Comment:
```suggestion
                    ->useAttributeAsKey('name')
                    ->variablePrototype()->end()
                ->end()
```
Having a configuration test or at least extension test would be nice